openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/RequestDescriptionPage/RequestDescriptionPage.tsx

```diff
@@ -125,7 +126,9 @@ const RequestDescription = () => {
           history.push(
             getEntityDetailLink(
               entityType as EntityType,
-              entityFQN,
+              entityType === EntityType.TABLE
+                ? entityFQN
+                : getDecodedFqn(entityFQN),
```

Comment:
@Ashish8689 Let's sync this encode / decode in 1.2

Reply:
yes!